packages/vscode-ide-companion/tsconfig.json

```diff
@@ -13,7 +13,9 @@
      */
     "skipLibCheck": true,
     "rootDir": "src",
-    "strict": true /* enable all strict type-checking options */
+    "strict": true /* enable all strict type-checking options */,
+    "skipLibCheck": true /* Helps avoid potential type conflicts between different third-party libraries in node_modules. */,
+    "types": ["vscode"]
```

Comment:
this seems like a fine change but unrelated. can you revert it from here and land it separately.